semanticdb/metap/src/main/scala/scala/meta/internal/metap/Main.scala

```diff
@@ -1,42 +1,55 @@
 package scala.meta.internal.metap
 
+import java.io.PrintStream
 import java.nio.file._
 import java.util.WeakHashMap
 import scala.collection.{immutable, mutable}
 import scala.compat.Platform.EOL
 import scala.math.Ordering
 import scala.util.control.NonFatal
 import scala.meta.internal.semanticdb3._
-import Diagnostic._, Severity._
-import SymbolInformation._, Kind._, Property._
-import SymbolOccurrence._, Role._
-import Type.Tag._, SingletonType.Tag._, Accessibility.Tag._
+import Diagnostic._
+import Severity._
+import SymbolInformation._
+import Kind._
+import Property._
+import SymbolOccurrence._
+import Role._
+import Type.Tag._
+import SingletonType.Tag._
+import Accessibility.Tag._
+import scala.Predef.{println => _, print => _, _}
```

Comment:
I suggest we avoid these shenanigans and explicitly say `out.print` and `out.println` everywhere. In my opinion, the necessity to unimport stuff from `Predef` suggests that a more straightforward solution would be preferable.

Reply:
Looks done. Thanks for the hard work!